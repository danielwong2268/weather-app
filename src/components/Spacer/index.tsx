import React from 'react';
import styled from 'react-emotion';
import { BLUE_1 } from 'src/constants/colors';

const HeaderContainer = styled('div')`
  height: 100px;
  background-color: ${BLUE_1};
`

interface HeaderProps {
  children?: React.ReactNode;
}

const Spacer = ({ children }: HeaderProps) => (
  <HeaderContainer>
    { children }
  </HeaderContainer>
)

export default Spacer;
