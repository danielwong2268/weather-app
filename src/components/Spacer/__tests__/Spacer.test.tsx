import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Spacer from '..';

describe('<Spacer />', () => {
  it('Renders with children', () => {
    const renderer = ShallowRenderer.createRenderer();
    renderer.render(<Spacer>child</Spacer>);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});