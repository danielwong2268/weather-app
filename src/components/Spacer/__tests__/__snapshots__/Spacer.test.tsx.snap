// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spacer /> Renders with children 1`] = `
<Styled(div)>
  child
</Styled(div)>
`;
